src/Signer/Ecdsa/Asn1.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Signer\Ecdsa;
+
+use FG\ASN1\ASNObject;
+use FG\ASN1\Universal\Integer;
+use FG\ASN1\Universal\Sequence;
+use InvalidArgumentException;
+use const STR_PAD_LEFT;
+use function assert;
+use function bin2hex;
+use function gmp_init;
+use function gmp_strval;
+use function hex2bin;
+use function mb_strlen;
+use function mb_substr;
+use function str_pad;
+
+final class Asn1 implements PointsManipulator
+{
+    public function fromEcPoint(string $signature, int $length): string
+    {
+        $sequence = ASNObject::fromBinary($signature);
+        assert($sequence instanceof Sequence);
+
+        $signature = '';
+
+        foreach ($sequence->getChildren() as $child) {
+            $signature .= str_pad($this->decToHex($child->getContent()), $length, '0', STR_PAD_LEFT);
+        }
+
+        return hex2bin($signature);
+    }
+
+    public function toEcPoint(string $points, int $length): string
+    {
+        $points = bin2hex($points);
+
+        if (mb_strlen($points, '8bit') !== 2 * $length) {
+            throw new InvalidArgumentException('The length of given value is different than expected');
+        }
+
+        $pointR = mb_substr($points, 0, $length, '8bit');
```

Comment:
Any kind of naming that would make this easier to understand, or is this from a reference implementation? If it is so, then let's please `@link` it

Reply:
This is essentially the logic [required by JWA](https://tools.ietf.org/html/rfc7518#page-90). I can add the link there.